from unittest import TestCase


class Test(TestCase):
    def test_enemy_attack(self):
        self.fail()
